packages/aws-cdk/lib/init-templates/app/dotnet/src/HelloCdk/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Amazon.CDK;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HelloCdk
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Create a new app. The first argument is used to display a usage message for this app.
+            string[] appArgs = new [] { $"dotnet ${nameof(HelloCdk)}" }
+                .Concat(args)
+                .ToArray();
+            var app = new App(appArgs);
```

Comment:
Any way we can make this more concise? The usage message is not viewed by anyone, I would just write "app" as argv0